ui/apps/platform/src/Containers/Policies/policies.utils.ts

```diff
@@ -157,23 +158,27 @@ export function formatEventSource(eventSource: PolicyEventSource): string {
 export function getExcludedDeployments(exclusions: PolicyExclusion[]): PolicyExcludedDeployment[] {
     const excludedDeploymentScopes: PolicyExcludedDeployment[] = [];
 
-    exclusions.forEach(({ deployment }) => {
-        if (deployment?.name || deployment?.scope) {
-            excludedDeploymentScopes.push(deployment);
-        }
-    });
+    exclusions
+        .filter((e): e is PolicyDeploymentExclusion => 'deployment' in e)
```

Comment:
Thank you, TypeScript!
https://devblogs.microsoft.com/typescript/announcing-typescript-5-5/#inferred-type-predicates

Reply:
Soon 🙏